xbmc/input/mouse/MouseStat.h

```diff
@@ -205,23 +205,23 @@ class CMouseStat
   bool MovedPastThreshold() const;
 
   // state of the mouse
-  MOUSE_STATE m_pointerState;
-  MouseState m_mouseState;
+  MOUSE_STATE m_pointerState{MOUSE_STATE_NORMAL};
+  MouseState m_mouseState{};
   bool m_mouseEnabled;
   CButtonState m_buttonState[MOUSE_MAX_BUTTON];
 
-  int m_maxX;
-  int m_maxY;
-  float m_speedX;
-  float m_speedY;
+  int m_maxX{0};
+  int m_maxY{0};
+  float m_speedX{0};
+  float m_speedY{0};
 
   // active/click timers
   unsigned int m_lastActiveTime;
 
-  bool bClick[MOUSE_MAX_BUTTON];
-  bool bDoubleClick[MOUSE_MAX_BUTTON];
-  HoldAction m_hold[MOUSE_MAX_BUTTON];
-  bool bLongClick[MOUSE_MAX_BUTTON];
+  bool bClick[MOUSE_MAX_BUTTON]{false};
+  bool bDoubleClick[MOUSE_MAX_BUTTON]{false};
+  HoldAction m_hold[MOUSE_MAX_BUTTON]{};
+  bool bLongClick[MOUSE_MAX_BUTTON]{false};
```

Comment:
This works by luck only, as this sets only the first element to false, all others will be filled with 0, which luckily is interpreted as false. 

But if you would like to default init all elements with `true`, this would not work with `bool bLongClick[MOUSE_MAX_BUTTON]{true};`!

Reply:
so you suggest to change to `{false}` with `{0}` ? set 0 is good for this use case

> BTW: why are those member's names not prefixed with m_? I thought this is coding guideline?

this is old code, my intention is only solve jenkins warnings
if you are asking to rewrite full class to follow guideline can be done in future

Reply:
I suggest use `{}` as is default initializer for any array type.

Reply:
> I suggest use {} as is default initializer for any array type.

Yes, as long as the array should be default initialized with `0`, which is the case here.

Reply:
no problem
done